portfolio/src/main/webapp/contact.html

```diff
@@ -24,6 +24,39 @@
         <p class = "phone"> phone: 727-417-8552</p>
     </div>
 
+    <div class= "form-comment-wrapper">
+
+    <form action="/data" method="POST" class = "comment-form">
+
+        <p class= "leave-a-comment"> Leave a comment!<p>
+        <textarea name="name" placeholder= "Name:"></textarea>
+        <br/><br/>
+        <textarea name="email" placeholder= "Email:"></textarea>
+        <br/><br/>
+        <textarea name="comment" placeholder="Comment:"></textarea>
+        <br/><br/>
+        <input type="submit" />
+    </form>
+    
+    <div class = "comment-choice-container">
+    <label class = "choose-comment-number">Choose how many comments you want to appear:</label>
+        <select name="comment-number" id="comment-number" onchange= "loadComments()">
+            <option value="5">5</option>
+            <option value="10">10</option>
+            <option value="15">15</option>
+            <option value="20">20</option>
+        </select>
+
+    <span id = "comment-container"> 
```

Comment:
I believe you're creating `div`s within this? I recently read something that you're techincally not supposed to put a `div` inside a `span`, though most browsers won't actually care.

Reply:
changed it to a div